includes/class-newspack-popups-segmentation.php

```diff
@@ -424,16 +508,22 @@ function( $segment ) {
 	/**
 	 * Create a segment.
 	 *
-	 * @param object $segment A segment.
+	 * @param object  $segment A segment.
+	 * @param boolean $get_id If true, return only the created segment ID. Otherwise, return all segments.
```

Comment:
It's weird that this method is returning all segments, and not the created segment. But refactoring it is maybe not worth the effort. 
Nit:
```suggestion
	 * @param boolean $return_id If true, return the created segment ID. Otherwise, return all segments.
```

Reply:
Updated the var name in debe5f6. I believe the method returns all segments because the Campaigns wizard UI expects all segments in the API response after creating a segment, and uses the response to re-render the segment list.

Reply:
Yes, that's the reason. It's just unintuitive when looking at the function itself. But that's a nitpick.